theano/sandbox/cuda/opt.py

```diff
@@ -1942,7 +1942,9 @@ def local_gpu_downsample_factor_max_grad_grad(node):
     if isinstance(node.op, pool.DownsampleFactorMaxGradGrad):
         assert node.op.__props__ == ('ds', 'ignore_border', 'st',
                                      'padding', 'mode')
-        if node.op.padding != (0, 0) or node.op.mode != 'max':
+        if (node.op.padding != (0, 0) or
+                node.op.mode != 'max' or
+                node.op.st != (1, 1)):
```

Comment:
Actually, I think this should be `node.op.st != node.op.ds`.
A fix is on the way once I check that it's correct.